index.html

```diff
@@ -33,19 +37,95 @@ <h1 class="logo">
         hesitate and contact me.
       </p>
 
-      <button type="button">Lets Connect</button>
+      <button type="button" class="connect">Lets Connect</button>
       <img class="design-one" src="bgbg.png" alt="hardwork bro" />
     </section>
 
     <section class="Projects">
       <div class="headingtwo">Projects</div>
-      <img src="Rectangle21.png" alt="project 1" />
+
       <div class="project-info">
```

Comment:
- [ ] Kindly make every `div` with class `project-info` have a `display: flex` according to project requirements. 👍